docs/elasticsearch.md

```diff
@@ -0,0 +1,20 @@
+# Exporting cAdvisor Stats to Elasticsearch
+
+cAdvisor supports exporting stats to [Elasticsearch](https://www.elastic.co/). To use ES, you need to provide the additional flags to cAdvisor:
+
+Set the storage driver as ES.
+
+```
+ -storage_driver=elasticsearch
+```
+
+Specify ES host address.
+
+```
+ -storage_driver_es_host="http://elasticsearch:9200"
+```
+
```

Comment:
Can you add a comment about the optional flags too:
```
	argTypeName      = flag.String("storage_driver_es_type", "stats", "ElasticSearch type name")
	argEnableSniffer = flag.Bool("storage_driver_es_enable_sniffer", false, "ElasticSearch uses a sniffing process to find all nodes of your cluster by default, automatically")
```
(The comment on the enable_sniiffer flag is confusing, since it's disabled by default)